FreeRTOS/FreeRTOS-Kernel
# Cortex-M Assert when NVIC implements 8 PRIO bits

Cortex-M Assert when NVIC implements 8 PRIO bits

Description
-----------
Maximum value of priority grouping is calculated incorrectly , when all 8 priority bits are implemented, which results in Assert.
This PR fixes this issue.

Test Steps
-----------
FreeRTOS_TCP_Echo_QEMU Demo implements 8 priority bits and works with these fixes.
GCC , IAR and RVDS ports compilation have been verified with the changes.

Checklist:
----------


- [ x ] I have tested my changes. No regression in existing tests.

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
